roles/boot_iso/tasks/nokia/main.yml

```diff
@@ -0,0 +1,62 @@
+---
+- name: Check boot iso is not in webroot
+  assert:
+    that:
+      - boot_iso_url is match("https?://.+/.+/.+")
```

Comment:
Do the nokia servers require the same `http://domain.com/foo/bar.iso` pattern as required by the SuperMicros if not lets not enforce it.